automation/src/main/java/org/wso2/testgrid/automation/reader/JMeterTestReader.java

```diff
@@ -65,20 +62,23 @@
                 Test test = new Test(scenario.getName(), TestEngine.JMETER, scriptList, scenario);
 
                 scripts.stream()
-                        .filter(x -> x.endsWith(SHELL_SUFFIX) && x.contains(PRE_STRING))
```

Comment:
These per scenario **_pre_** and **_post_** scripts are not being used now, hence IMO we could remove the TestReading logic from here WDYT?

Reply:
+1. Can be handled outside of this pr.

Reply:
https://github.com/wso2/testgrid/issues/830